website/docs/docs/get-started/privatelink/redshift-privatelink.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Configuring Redshift PrivateLink"
+id: redshift-privatelink
+description: "Configuring PrivateLink for Redshift"
+sidebar_label: "Redshift PrivateLink"
+---
+
+## Configuring managed PrivateLink
+
+The first step is to grant access to dbt Cloud in the Redshift database cluster:
+
+1. On the running Redshift cluster select the **Properties** tab.
+
+<Lightbox src="/img/docs/dbt-cloud/redshiftprivatelink1.png" title="Redshift Properties tab"/>
+
+2.Scroll to the **Granted accounts** section and click the **Grant access** button.
+
+<Lightbox src="/img/docs/dbt-cloud/redshiftprivatelink2.png" title="Redshift granted accounts"/>
+
+3. Enter the AWS account ID:
+
+- dbt Cloud multi-tenant `346425330055` 
+- dbt Cloud single-tenant `952445382732`
+
+Optionally, provide the VPC ID for the single-tenant instance or regional multi-tenatn PrivateLink VPC, otherwise choose Grant access to all VPCs.
+
+<Lightbox src="/img/docs/dbt-cloud/redshiftprivatelink3.png" title="Redshift grant access"/>
+
+Once those steps are completed, please contact the dbt support team and provide them with:
+
+- Cluster name
+- AWS account ID (where the Redshift cluster is hosted)
+- Cluster AWS region
+
+## Configuring Interface VPC PrivateLink
+
+### Provisioning the VPC
+
+Creating an Interface VPC PrivateLink connection requires creating multiple AWS resources in the account containing the Redshift cluster:
+
+- **Security Group** &mdash; If connecting to an existing Redshift cluster this likely already exists, however Security Group rules may need to be added or modified to accept traffic from the Network Load Balancer (NLB) created for this Endpoint Service.
+- **Target Group** &mdash; The Target Group gets attached to the NLB to tell it where to route requests. There are various target types available for NLB Target Groups, but you will use IP address.
+
```

Comment:
if this paragraph is telling the user to do something, consider using more second-person words like "When you create an Interface VPC PrivateLink connection, you need to create .... "

i think it's good to be clear since dbt support is also performing things on behalf of the user